packages/snage/src/command/common.ts

```diff
@@ -12,6 +21,20 @@ export const printAndExit = (err: string): IO.IO<never> => () => {
     process.exit(1);
 };
 
-export const print = (msg: string): IO.IO<void> => () => {
-    console.log(msg);
-};
+export const print = (msg: string): IO.IO<void> => () => console.log(msg);
+
+export const parseNotesWithExpression = (config: Config, condition: string): TE.TaskEither<string, Note[]> =>
```

Comment:
I can see that you use this method in multiple places, but it feels a bit like a weird abstraction.
Can we simplify the implementation to something like
```typescript
pipe(
  parseNotes(config),
  TE.chainEitherK(filterNotes(condition))
)
```
and inline the method again?